Automattic/wp-calypso
# add: Jetpack support warning for non hosted sites

#### Changes proposed in this Pull Request

* This PR aims to add more context to our "Contact us" form for customers with self-hosted WordPress sites.
More context: p7pQDF-6HR-p2

#### Testing instructions

* Go to http://calypso.localhost:3000/help/contact
* Select a self hosted site
* You should be able to see the warning:
![Screenshot 2022-04-11 at 10 43 28](https://user-images.githubusercontent.com/8320845/162712100-49bcb100-b6e3-4824-9640-5596432dbd5c.png)

It's also good to retest the previous warning message added in this PR: https://github.com/Automattic/wp-calypso/pull/52979

Please let me know if you find any issues or have any suggestions
